client/src/features/Communication/components/AddParticipants.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState, useEffect } from "react";
+import CommunicationServiceAPI from "../../../api/communicationServiceAPI";
+import UserServiceAPI from "../../../api/userServiceAPI.ts";
+import { Dropdown } from "react-bootstrap"; // Import Bootstrap Dropdown
+import Button from "react-bootstrap/Button"; // Import Bootstrap Button
+import Modal from "react-bootstrap/Modal"; // Import Bootstrap Modal
+
+const AddParticipantModal = () => {
+  const [userId, setUserId] = useState("");
+  const [employeeOptions, setEmployeeOptions] = useState([
+    { key: "", text: "" },
+  ]);
+  const [loading, setLoading] = useState(true);
+  const [showModal, setShowModal] = useState(false);
+
+  useEffect(() => {
+    const fetchEmployeeOptions = async () => {
+      try {
+        const employees = await UserServiceAPI.getInstance().usersInCompany();
+        const userIds = employees.map((user) => ({
+          key: user.user_id.toString(),
+          text: `${user.first_name} ${user.last_name}`,
+        }));
+        setEmployeeOptions(userIds);
+        setLoading(false);
+      } catch (error) {
+        console.error("Error fetching employee options:", error);
+      }
+    };
+
+    fetchEmployeeOptions();
+  }, []);
+
+  const handleChatButtonClick = async () => {
+    let conversationId;
+    try {
+      conversationId = "54321";
```

Comment:
This appears to be hardcoded, is this temporary or needed?

Reply:
This will be fixed in a later pr to remove the hardcoded value but for now it is needed.